project-tracker.njsproj

```diff
@@ -27,8 +27,8 @@
     <Content Include="public\index.html" />
     <Compile Include="app\models\todo.js" />
     <Compile Include="public\js\core.js" />
-    <Compile Include="public\js\controllers\main.js" />
-    <Compile Include="public\js\services\todo.services.js" />
+    <Compile Include="public\js\controllers\main.controller.js" />
```

Comment:
Well done, good practice use angular type in name of file. This is much easier to find it in solution